donation-ui/donation-ui/src/app/dashboard/dashboard.component.css

```diff
@@ -0,0 +1,107 @@
+.bg-color{
```

Comment:
please for the frontEnd structure, follow this one: https://github.com/adamip/DeborahK_Angular2_GettingStarted/tree/master/APM%20-%20Final/app
separation by feature and section not by (controller-model-service) structure